pcs/src/toeplitz.rs

```diff
@@ -30,6 +31,7 @@ pub struct CirculantMatrix<F: FftField> {
 
 impl<F: FftField> CirculantMatrix<F> {
     /// Construct a Circulant matrix by its first column vector
+    #[allow(dead_code)]
```

Comment:
this is a public api, should be fine if unused i believe?

Reply:
removed [ac0c37b](https://github.com/EspressoSystems/jellyfish/pull/556/commits/ac0c37bcb8daaef34ea55be9e6456d78f8359216)